step/src/main/java/com/google/sps/servlets/InfoServlet.java

```diff
@@ -53,7 +53,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String email = user.getEmail();
     String id = user.getUserId();
     
-    DatabaseHandler.addUser(firstName, lastName, dayBirth, monthBirth, yearBirth, email, id);
+    // Ensuring that only users who have filled all of the info fields out are added to the database
```

Comment:
I would also probably limit people under the age of 13 as per the Children's Online Privacy Protection Act. This can be a separate change but something to keep in mind.